docs/users/extend/customization-extendibility.md

```diff
@@ -13,6 +13,24 @@ A collection of vetted service configurations is available in the [Additional Se
 
 If you need to create a service configuration for your project, see [Defining an additional service with Docker Compose](custom-compose-files.md)
 
+## Providing custom environment variables to a container
+
+Each project can have an unlimited number of .ddev/docker-compose.*.yaml files, so it's easy to maintain custom environment variables in a .ddev/docker-compose.environment.yaml file (the exact name doesn't matter, if it just matches docker-compose*.y*ml).
```

Comment:
nit, the last mentioned file name format should be docker-compose.*.y*ml. the first mentioned one also uses .yaml vs .y*ml. i wonder if we should just document one of the two yaml extensions rather than using the *. not sure how much confusion that could be for some people